src/main/resources/js/jstree/jstree.js

```diff
@@ -26,7 +26,7 @@
     console.log($.fn.jquery);
     
     console.log("window.$ ver");
-    console.log(window.$.fn.jquery);
+    //console.log(window.$.fn.jquery); // $ is not found because we add <context>jira.view.issue</context> in the atlassian-plugin.xml
     
```

Comment:
same as above:
This line and the line above should be just removed. The source code comment `// $ is not found ...` could be moved to the commit message.